src/main/webapp/basic-search.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { connect } from 'react-redux'
+
+import Button from '@material-ui/core/Button'
+import Card from '@material-ui/core/Card'
+import CardContent from '@material-ui/core/CardContent'
+import TextField from '@material-ui/core/TextField'
+
+import { executeQuery } from './intrigue-api/lib/cache'
+
+const TextSearch = props => {
+  const [filterTree, setFilterTree] = React.useState({
+    type: 'ILIKE',
+    property: 'anyText',
+    value: '',
+  })
+
+  return (
+    <CardContent>
+      <TextField
+        label="Text"
+        variant="outlined"
+        fullWidth
+        value={filterTree.value}
+        onChange={e => {
+          const value = e.target.value
+          setFilterTree({ ...filterTree, value })
+        }}
+      />
+
+      <Button
+        fullWidth
+        variant="contained"
+        color="primary"
+        onClick={() => {
+          props.onSearch(filterTree)
+        }}
+      >
+        Search
+      </Button>
+    </CardContent>
+  )
+}
+
+const populateDefaultFilterTree = filterTree => ({
+  // hello source to test error case
+  srcs: ['ddf.distribution', 'cache', 'hello'],
```

Comment:
Do we want to add Error cases in our actual code, or would this be better to do in a Story?

Reply:
The error case was to test the error action and reducer and the Story isn't using the redux connected BasicSearch.
I'll remove it.